src/PermissionServiceProvider.php

```diff
@@ -123,6 +123,16 @@ protected function registerBladeExtensions($bladeCompiler): void
         $bladeCompiler->directive('elserole', fn ($args) => "<?php elseif({$bladeMethodWrapper}('hasRole', {$args})): ?>");
         $bladeCompiler->directive('endrole', fn () => '<?php endif; ?>');
 
+        $bladeCompiler->directive('haspermissionto', function ($arguments) {
+            return "<?php if(\\Spatie\\Permission\\PermissionServiceProvider::bladeMethodWrapper('hasPermissionTo', {$arguments})): ?>";
```

Comment:
Why not follow the syntax of the other directives?

Reply:
I tried to follow that syntax. Would you recommend `@haspermission` or another one better?

Reply:
When i say sintax, means PHP Sintax, not naming
```php
$bladeCompiler->directive('haspermissionto', fn ($args) => "<?php if({$bladeMethodWrapper}('hasPermissionTo', {$args})): ?>");
```

Reply:
you forgot `$args` `$bladeMethodWrapper`